go.mod

```diff
@@ -88,6 +88,7 @@ require (
 replace (
 	github.com/docker/distribution => github.com/docker/distribution v2.8.3+incompatible
 	github.com/docker/docker => github.com/docker/docker v25.0.5+incompatible
+	github.com/hashicorp/consul/api => github.com/hashicorp/consul/api v1.30.0
```

Comment:
Can we bump viper to 1.19 instead? This is no longer a dependency in that version.

Reply:
Ah, nevermind we'd have to go to a 1.20 alpha version for that.